### PR TITLE
[TextDraw] - No instance of StringBuilder needed here.

### PR DESCRIPTION
**Pull request by ivandrofly**

### CODE REVIEW

File: libse/TextDraw.cs [hunk 1]
```diff
@@ -100,11 +98,10 @@ public static float MeasureTextHeight(Font font, string text, bool bold)
             {
                 using (var path = new GraphicsPath())
                 {
-                    var sb = new StringBuilder(text);
```
Comment (ivandrofly) [inline libse/TextDraw.cs]: ☝️

File: libse/TextDraw.cs [hunk 2]
```diff
@@ -64,11 +63,10 @@ public static float MeasureTextWidth(Font font, string text, bool bold)
...
```
Comment (ivandrofly) [inline libse/TextDraw.cs]: ☝️

File: libse/TextDraw.cs [hunk 3]
```diff
@@ -6,7 +6,7 @@ namespace Nikse.SubtitleEdit.Core
 {
     public static class TextDraw
     {
-        public static void DrawText(Font font, StringFormat sf, GraphicsPath path, StringBuilder sb, bool isItalic, bool isBold, bool isUnderline, float left, float top, ref bool newLine, float leftMargin, ref int pathPointsStart)
+        public static void DrawText(Font font, StringFormat sf, GraphicsPath path, string s, bool isItalic, bool isBold, bool isUnderline, float left, float top, ref bool newLine, float leftMargin, ref int pathPointsStart)
```
Comment (ivandrofly) [inline libse/TextDraw.cs]: We can simply pass a string since no build operation is been done inside DrawText.

**Comment by niksedk** [comment]
If this is still relevant please add a new pr with info about why, what and unit tests

**Pull request closed without merging**